File: p1/p1.pl
#!/usr/bin/perl
use strict;
use warnings;

my $sum = 0;
for (my $i = 1; $i < 1000; $i++) {
    if ($i % 3 == 0 || $i % 5 == 0) {
        $sum += $i;
    }
}

print $sum;

File: p1/p1.rs
fn main() {
    let mut i: int = 1, sum: int = 0;
    while i < 1000 {
        if i % 3 == 0 || i % 5 == 0 {
            sum += i;
        }
        i += 1;
    }
    io::println(fmt!("%?", sum))
}
